src/platform_impl/macos/menu.rs

```diff
@@ -607,3 +622,18 @@ impl Accelerator {
     flags
   }
 }
+
+fn get_app_name() -> Option<&'static str> {
```

Comment:
there is a similar utility function in `src/plagrom_impl/macos/util/mod.rs`

Reply:
Huh, I looked and didn't find anything equivalent, but I'll have another look later

Reply:
I found the function you're referring to but it doesn't seem to work, at least in the example apps. I think the problem is it's only looking in the `Info.plist` file whereas the API I used also checks other sources.

Since that function isn't used anywhere in Tao, can I modify the function signature? If possible I'd like to change the return type to `Option<&str>` and also make it safe. It [doesn't seem to be exported publicly](https://docs.rs/tao/latest/tao/?search=app_name) so I guess it won't be a breaking change – right?